src/safeposix/syscalls/net_calls.rs

```diff
@@ -1765,22 +1765,102 @@ impl Cage {
         return 0;
     }
 
+    /// ### Description
+    /// 
+    /// `accept_syscall` accepts a connection on a socket 
+    /// 
+    /// ### Arguments
+    /// 
+    /// it accepts two parameters: 
+    /// * `fd` - the file descriptor that refers to the listening socket
+    /// * `addr` - the address of the peer socket (i.e. the client that is connecting)
```

Comment:
might be pedantic, but do you think using connection directionality would be clearer? For example "the address of the peer socket (i.e. the address of the incoming connection)"